test/extensions/filters/http/fault/fault_filter_test.cc

```diff
@@ -588,6 +588,86 @@ TEST_F(FaultFilterTest, FixedDelayDeprecatedPercentAndNonZeroDuration) {
   EXPECT_EQ(0UL, config_->stats().active_faults_.value());
 }
 
+// Verifies that 2 consecutive delay faults be allowed with the max number of faults set to 1
```

Comment:
Sorry, should have added a comment before, but can you add this as a bug fix in 
docs/root/version_history/current.rst?

Reply:
Sure, will do. Thanks.